tests/ui/test_error_improvements.py

```diff
@@ -0,0 +1,131 @@
+import logging
+
+from ocs_ci.framework.pytest_customization.marks import (
+    ui,
+    skipif_ibm_cloud_managed,
+    skipif_managed_service,
+    black_squad,
+    polarion_id,
+)
+from ocs_ci.framework.testlib import ManageTest
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.ui.base_ui import PageNavigator
+
+logger = logging.getLogger(__name__)
+
+
+@ui
```

Comment:
as discussed, please replace `@ui` with tier3

Reply:
can we have both ui and tier3? or better to remove ui?